### DISCUSSION TITLE
Attach Model to Bone?   Code-Only

### DISCUSSION THREAD

**Discussion by najak3d** [stride3d/stride#1352] [Q&A] [answered]
I see the editor method has a link for how to do this, here:
https://doc.stride3d.net/4.0/en/manual/animation/model-node-links.html

I'd like to request a demo for how this should be done with the Code-Only approach, or perhaps it's a combination effort - that you set it up in the Editor (bones names, default rotation/transforms), but then do the attach/detach operations from code.

I'm guessing this might be a fairly easy request.   I'm a noob at Stride3D, but am planning to port our current project from Urho3D.NET over to Stride3D in the near future.

**Comment by Eideren** [comment, marked as answer]
```cs
myToolEntity.Add(new ModelNodeLinkComponent(){ Target = myCharacterModelComponent, NodeName = "RightHand" });
```

**Reply by najak3d** [reply]
Looks simple and clear.   Stride3D wins again.
